README.md

```diff
@@ -1 +1,40 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Webpage
+
+> My personal portfolio webpage (under development).
+
+![screenshot](./screenshot.png)
+
+Webpage I use to show my work and selling my services.
+
+## Built With
+
+- HTML&CSS
+
+## Getting Started
+
+**How to use.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Web browser (Google Chrome, Firefox, Safari)
+### Setup
+git clone the repository
+### Usage
+Open the file "index.html" with your browser
+### Run tests
+cd into the cloned folder and run npx hint. and npx stylelint "**/*.{css,scss}"
+
+## Authors
+
+👤 **Fabrizio**
+
+- GitHub: [@githubhandle](https://github.com/KatIsCoding)
+- Twitter: [@twitterhandle](https://twitter.com/fabriziogr211)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fabrizio-gomez-6a00801a3/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
Invitation to go to the issue page is missing, it can help you to see your README file more professional.

- [ ] Add Invitation to go to the issue page if there are errors in your repository, try to see how the [Template provided by Microverse](https://github.com/microverseinc/readme-template) can help you to do that.
- [ ] Add a License for your project, also you can check it out in the [Template provided by Microverse](https://github.com/microverseinc/readme-template).